roles/rsyslog/tasks/set_vars.yml

```diff
@@ -7,6 +7,21 @@
   when: __logging_required_facts |
     difference(ansible_facts.keys() | list) | length > 0
 
+- name: Determine if system is booted with systemd
+  when: not __logging_is_booted is defined
+  block:
+    - name: Run systemctl
+      # noqa command-instead-of-module
+      command: systemctl is-system-running
+      register: __is_system_running
+      changed_when: false
+      failed_when: false
```

Comment:
I would prefer not to use `failed_when: false` because it masks all errors - would it be possible to do something like `failed_when: not __is_system_running.rc in [0, 1]` assuming that `systemctl is-system-running` returns an error code of `1` when `__is_system_running.stdout` is `offline` or `degraded`?

Reply:
Unfortunately not -- the [manpage](https://www.man7.org/linux/man-pages/man1/systemctl.1.html#:~:text=is-system-running%20output) only guarantees "exit code > 0". It seems to be 0 *right now*, but TBH I'd rather not rely on that.